lib/identity_config.rb

```diff
@@ -253,6 +253,8 @@ def self.build_store(config_map)
     config.add(:risc_notifications_rate_limit_max_requests, type: :integer)
     config.add(:risc_notifications_rate_limit_overrides, type: :json)
     config.add(:ruby_workers_enabled, type: :boolean)
```

Comment:
Since we now have workers enabled by default in all our envs, and things like cronjobs are guarded on a feature flag, I would consider maybe removing this old `ruby_workers_enabled` flag? Open to feedback on it

Reply:
I think it might make sense to remove it? Unless we want the double safety of having to enable a couple flags, but my worry is it may be more confusing than it's worth.

Reply:
Gave it a shot in f801a278649ddac0c657c297e8b011d2fec858c2

Reply:
Once this merges, I'll make a pass to update the handbook docs too